src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -40,6 +41,25 @@ export interface Manifest {
   endpoints: Record<string, ManifestEndpoint>;
 }
 
+/** Returns a Build from a v1alpha1 Manifest. */
+export function buildFromV1Alpha1(
+  yaml: unknown,
+  project: string,
+  region: string,
+  runtime: runtimes.Runtime
+): build.Build {
+  const manifest = JSON.parse(JSON.stringify(yaml)) as Manifest;
+  const bd: build.Build = build.empty();
```

Comment:
TODO: consider adding the following check as done in `backendFromV1Alpha1`.

```
  assertKeyTypes("", manifest, {
    specVersion: "string",
    requiredAPIs: "array",
    endpoints: "object",
  });
```

Reply:
Done.